sdk/mixedreality/azure-mixedreality-remoterendering/azure/mixedreality/remoterendering/_remote_rendering_client.py

```diff
@@ -0,0 +1,350 @@
+from azure.mixedreality.authentication._shared.static_access_token_credential import \
+    StaticAccessTokenCredential
+
+from ._generated import RemoteRenderingRestClient
+# TODO: do i need these for type checking?
+from ._generated.models import (Conversion, ConversionSettings,
+                                ConversionStatus, CreateConversionSettings,
+                                CreateSessionSettings, SessionProperties,
+                                SessionSize, SessionStatus,
+                                UpdateSessionSettings)
+from ._version import SDK_MONIKER
+
+if TYPE_CHECKING:
+    # pylint: disable=unused-import,ungrouped-imports
+    from typing import Any, Callable, Union
+
+    from azure.core.credentials import TokenCredential
+
+
+# TODO: factor out/check if this is really the right way to go
+# taken from: \sdk\communication\azure-communication-administration\azure\communication\administration\_polling.py
+class RemoteRenderingPolling(PollingMethod):
+    """ Abstract base class for polling.
+    """
+
+    def __init__(self, account_id, is_terminated, polling_interval=5):
+        # type: (str, Callable, int) -> None
+        self._account_id = account_id
+        self._response = None  # type: Union[Conversion, SessionProperties, None]
+        self._client = None  # type: Union[RemoteRenderingClient, None]
+        self._query_status = None
+        self._is_terminated = is_terminated
+        self._polling_interval = polling_interval
+
+    def _update_status(self):
+        # type: () -> None
+        if self._query_status is None:
+            raise Exception("this poller has not been initialized")
+        self._response = self._query_status()  # pylint: disable=E1102
+
+    def initialize(self, client, initial_response, deserialization_callback):
+        # type: (Any, Any, Callable) -> None
+        self._client = client
+        self._response = initial_response
+
+    def run(self):
+        # type: () -> None
+        while not self.finished():
+            self._update_status()
+            if not self.finished():
+                time.sleep(self._polling_interval)
+
+    def finished(self):
+        # type: () -> bool
+        if self._response is None:
+            return False
+        if self._response.status is None:
+            return False
+        return self._is_terminated(self._response.status)
+
+    def resource(self):
+        # type: () -> Union[Conversion, SessionProperties, None]  #TODO: fix type annotation - to i need ~...?
+        if not self.finished():
+            return None
+        return self._response
+
+    def status(self):
+        # type: () -> str
+        if self._response is None:
+            return "Failed" # TODO: is this smart? Session and Conversion do not share status
+        return self._response.status
+
+    def get_continuation_token(self):
+        # type() -> str
+        # TODO: rikogeln what is the continuation token for
```

Comment:
does the implementation make sense like this? for us the conversion id would be enough information to continue polling (if we simply call get_asset_conversion()) - would this be a better option?

Reply:
The rehydration of a poller is a pattern we are still trying to nail down across our SDKs....
In the meantime we should make sure this scenario is unblocked.
Given that you have a custom polling method - you can have `get_continuation_token` return whatever you need to restart the poller (though we have typically included the serialized initial state, so the poller has a "starting point").
What we'll need to look at is what a customer is supposed to do with the continuation_token/conversion_id. In the meantime for this preview -  you could add a `from_continuation_token`  kwargs/keyword-only parameter to `begin_asset_conversion` to allow customers to get the poller for an ongoing conversion while we figure out a more user-friendly pattern for this.

Reply:
@annatisch - i was about to implement this, but what am i supposed to do with the mandadory parameters like input_settings, output_settings etc in case we just want to continue polling. 
i added a get_asset_coversion_poller and get_rendering_session_poller method which can get a continuation token or id ... not happy about the naming. any suggestion?